relayer/canonical_validator_client.go

```diff
@@ -8,15 +8,20 @@ import (
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/vms/platformvm"
+	"go.uber.org/zap"
 )
 
+var _ validators.State = (*CanonicalValidatorClient)(nil)
```

Comment:
nit: I think the syntax `&CanonicalValidatorClient{}` on the rhs is a bit cleaner, and it's also what we've used in other places in our repos.

Reply:
Sounds good to me. There was one other place in this repo that we were using the `(*Type)(nil)` syntax, so I updated it there as well 👍